web/src/pages/core/public/Lectures/Lectures.jsx

```diff
@@ -31,45 +35,62 @@ const Lectures = () => {
-                {field: 'post_time', headerName: 'Post Time', width: 170},
-                {field: 'title', headerName: 'Title', width: 300},
-                {
-                  field: 'a', headerName: 'Lecture Attachment', width: 300, renderCell: ({row}) => (
-                    <div>
-                      <Typography
-                        variant={'body1'}
-                        color={'primary'}
-                        style={{display: 'inline'}}
-                        component={'a'}
-                        target={'_blank'}
-                        href={get_href(row)}
-                      >
-                        {row.static_file.filename}
-                      </Typography>
-                    </div>
-                  ),
-                },
-              ]}
-              rows={lectures}
-              rowsPerPageOptions={[10, 20, 30]}
-              pagination
-            />
-          </Paper>
-        </Grid>
-      </Grid>
+    <StandardLayout>
+      <SectionHeader isPage title = 'Lectures'/>
+      <Divider />
+      <ListHeader sections = {['Course', 'Post Time', 'Title', 'Lecture Attachment']} />
+      {lectures.map((lecture, index) => (
+        <LectureItem
+          key = {`${lecture.name}-${index}`}
+          course = {lecture.course}
+          postTime = {lecture.post_time}
+          title = {lecture.title}
+          id = {lecture.id}
+          fileAttachment = {get_href(lecture)}
+        />
+      ))}
     </StandardLayout>
   );
 };
+//     <StandardLayout title={'Lectures'} description={'Lecture notes posted by professors'}>
+//       <Grid container spacing={1} justify={'center'}>
+//         <Grid item xs={12}>
+//           <Paper className={classes.paper}>
+//             <DataGrid
+//               columns={[
+//                 {field: 'course', headerName: 'Course', width: 150},
+//                 {field: 'post_time', headerName: 'Post Time', width: 170},
+//                 {field: 'title', headerName: 'Title', width: 300},
+//                 {
+//                   field: 'a', headerName: 'Lecture Attachment', width: 300, renderCell: ({row}) => (
+//                     <div>
+//                       <Typography
+//                         variant={'body1'}
+//                         color={'primary'}
+//                         style={{display: 'inline'}}
+//                         component={'a'}
+//                         target={'_blank'}
+//                         href={get_href(row)}
+//                       >
+//                         {row.static_file.filename}
+//                       </Typography>
+//                     </div>
+//                   ),
+//                 },
+//               ]}
+//               rows={lectures}
+//               rowsPerPageOptions={[10, 20, 30]}
+//               pagination
+//             />
+//           </Paper>
+//         </Grid>
+//       </Grid>
+//     </StandardLayout>
+//   );
+// };
```

Comment:
Please remove the comments, from the pr that arent needed. I you are ever curious about a previous state you can look at the history.